src/leases/components/LeasePage.tsx

```diff
@@ -261,6 +266,7 @@ class LeasePage extends Component<Props, State> {
 
     if (prevProps.currentLease !== currentLease) {
       this.setPageTitle();
+      fetchReceivableTypes();
```

Comment:
No, because, these receivable types are only related to the lease's receivable type. The set of available receivable types should not change depending on the active service unit.